CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required( VERSION 3.0 )
+
+project( cubature )
+
+set( CMAKE_C_FLAGS " -fPIC ${CMAKE_C_FLAGS}" )
+
+find_package( PkgConfig REQUIRED )
+pkg_check_modules( PKG_FFTW REQUIRED fftw3l )
```

Comment:
FFTW isn't required, because the generated file `clencurt.h` is included in the git repo.